### PR TITLE
[Simsala] automatic release created for v1.0.0-beta.110

### PR DESCRIPTION
**Pull request by faboweb**
Please manually test before merging this to master <a href='https://13243-99653950-gh.circle-artifacts.com/0/tmp/voyager/dist/index.html'>here</a>

### Repository

- this is needed for babel and jest to play nicely on my machine @jbibla

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = {
     "^.+\\.vue$": "vue-jest",
     ".+\\.(css|styl|less|sass|scss|svg|png|jpg|ttf|woff|woff2)$":
       "jest-transform-stub",
-    "^.+\\.jsx?$": "babel-jest"
+    "^.+\\.(js|jsx)?$": "babel-jest"
```
Comment (faboweb) [inline jest.config.js]: @jbibla I needed to remove this again as it broke my develop.
This expression also means: Any file with the ending : `.js`, `.jsx` or `.`. I think that is not intended.

Comment (jbibla) [reply jest.config.js]: saw your fix on the other branch. no problem.

Comment (jbibla) [inline jest.config.js]: you don't want this included, right?

Comment (faboweb) [reply jest.config.js]: better not. but will be overwritten by the mobile fixes anyway.

**Pull request merged**